release-notes/release-notes509.md

```diff
@@ -71,4 +74,38 @@ For more information on **Enhanced Media Embed** refer to the [documentation]({{
 
 **LinkChecker** 2.0.3 has addressed an issue where empty or blank links were incorrectly sent for validation and marked as invalid.
 
-For more information on **Link Checker** refer to the [documentation]({{site.baseurl}}/plugins/linkchecker/).
\ No newline at end of file
+For more information on **Link Checker** refer to the [documentation]({{site.baseurl}}/plugins/linkchecker/).
+
+
+## Errata&#58; TinyMCE 5.0.9 Premium Plugins
+
+This errata documents updates to the TinyMCE 5.0.9 Premium Plugins released on the 17th July, 2019. Future releases of TinyMCE will also include these changes.
+
+## Tiny Comments
```

Comment:
This and the PowerPaste heading should be a third level heading (eg `###`), as it's a subsection for the errata.

Reply:
Amended Commit to fix issue.